tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -1419,11 +1419,11 @@ def test_bulk_create_returns_rollback_response_on_bulk_index_exception(test_data
     log = LogEntry.objects.get()
     assert log.change_message == "Encountered error while indexing datafile documents: indexing exception"
 
-    assert all_created is False
-    assert len(unsaved_records.items()) == 3
+    assert all_created is True
+    assert len(unsaved_records.items()) == 0
     assert TANF_T1.objects.all().count() == 1
-    assert TANF_T2.objects.all().count() == 0
-    assert TANF_T3.objects.all().count() == 0
+    assert TANF_T2.objects.all().count() == 1
+    assert TANF_T3.objects.all().count() == 1
 
```

Comment:
can you add a test for cases that have bulkcreate error and assert the error?

Reply:
this is the test for the `BulkCreateError`. i only had to modify the existing test - this shows that the `LogEntry` for the error is still created, but instead of rolling back the created records they are committed to the db.

Reply:
Are we able to add a check that the docs weren't actually created?

Reply:
i didn't because that would really only check the patch of `TANF_T1DataSubmissionDocument.update`